cmd/odf/mon/mon.go

```diff
@@ -0,0 +1,18 @@
+package mon
+
+import (
+	"github.com/spf13/cobra"
+)
+
+// MonsCmd represents the mon command
+var MonsCmd = &cobra.Command{
+	Use:                "mons",
+	Short:              "Supports sub-commands like 'restore-quorum'",
```

Comment:
I'd suggest not to mention specific subcommands, just keep it general
```suggestion
	Short:              "Supports sub-commands for mons'",
```

Reply:
got it